commerce/bookstore/python/requirements.txt

```
@@ -0,0 +1,3 @@
+uuid
+argparse==1.4.0
+typedb-client
```

Let's give each requirement a version range to ensure that this project stays correct.
```suggestion
uuid>=1.30,<2
argparse>=1.4.0,<2
typedb-client>=2.12.1,<2.13
```
(Version ranges on `uuid` and `argparse` assume that they adhere to a common standard of not introducing breaking changes without a major version upgrade; version range on `typedb-client` is based on our own Vaticle versioning policy)